source/testers/fixtures.js

```diff
@@ -24,7 +24,7 @@ function normalizer() {
 }
 
 function cleanRenderedHtml(html) {
-  return (html ? normalizer().domString(html.trim()) : '');
+  return (html.trim() ? normalizer().domString(html.trim()) : '');
```

Comment:
If you could create a use-case for that inside `test` folder, that would be appreciated.